docs/cmd/kn_source_apiserver_create.md

```diff
@@ -0,0 +1,43 @@
+## kn source apiserver create
+
+Create a Kubernetes API Server event source
+
+### Synopsis
+
+Create a Kubernetes API Server event source
+
+```
+kn source apiserver create NAME --resources RESOURCE --service-account ACCOUNTNAME --sink SINK --mode MODE [flags]
+```
+
+### Examples
+
+```
+
+  # Create a Kubernetes API Server event source 'k8sevents' which consumes Kubernetes events and send them to service 'mysvc' as a cloudevent
+  kn source apiserver create k8sevents --resources Event:v1:true --service-account myaccountname --sink svc:mysvc
+```
+
+### Options
+
+```
+  -h, --help                     help for create
+      --mode string              'Ref' or 'Resource', the receive adapter controller runs under. (default "Ref")
```

Comment:
Could you briefly explain here the difference between `Ref` and `Resource` ?

Reply:
done.